app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialFragment.java

```diff
@@ -120,9 +130,11 @@
       actionBar.setDisplayHomeAsUpEnabled(true);
     }
     backArrow = toolbar.getNavigationIcon();
+    scrollView = (NestedScrollView) view.findViewById(R.id.nested_scroll_view);
     appBarLayout = (AppBarLayout) view.findViewById(R.id.app_bar_layout);
```

Comment:
I have to add the comment here because i can not comment on where the configuration of the listener for this layout is, but I would recommend extracting the configuration of the listener to another method just to improve code readibility (like configureAppBarLayout() )